addons/web_editor/static/src/js/common/utils.js

```diff
@@ -53,12 +53,19 @@ const DEFAULT_PALETTE = {
  * returns the cached computed value.
  *
  * @param {boolean} [toRem=false]
+ * @param {JQuery} $target The jquery element to base the style on.
  * @returns {float} - number of px by rem if 'toRem' is false
  *                  - the inverse otherwise
  */
-function _computePxByRem(toRem) {
+function _computePxByRem($target, toRem=false) {
     if (_computePxByRem.PX_BY_REM === undefined) {
-        const htmlStyle = window.getComputedStyle(document.documentElement);
+        let htmlStyle;
+        if ($target && $target.length) {
+            const $targetDocument = $target[0].ownerDocument;
+            htmlStyle = $targetDocument.defaultView.getComputedStyle($targetDocument.documentElement);
+        } else {
+            htmlStyle = window.getComputedStyle(document.documentElement);
+        }
```

Comment:
**IDEALLY BEFORE MERGE** (to avoid diff on existing code): I would like (in the future) to get rid of the fact `UserValueWidget` instances know about `$target`. I understand the need here but there are other ways to find the right window/document (the editor should somehow declare it so the utils work with it). cc @detrouxdev